config/helm/kustomization.yaml

```diff
@@ -4,13 +4,13 @@ resources:
   - input/cluster-api-components.yaml
 
 images:
-  - name: gcr.io/k8s-staging-cluster-api/cluster-api-controller
+  - name: k8s.gcr.io/cluster-api/cluster-api-controller
     newName: "{{ .Values.images.core.name }}"
```

Comment:
Here we don't specify Docker repository domain, so it uses Docker Hub by default with our mirroring config working.

Reply:
I meant that I thought we needed to leave the image as `cluster-api/cluster-api-controller` without the `k8s.gcr.io` part, because we needed to drop the domain and `k8s.gcr.io` was the domain.